src/main/java/org/folio/circulation/resources/OverrideRenewalByBarcodeResource.java

```diff
@@ -36,15 +38,20 @@ private void renew(RoutingContext routingContext) {
     final LoanRepository loanRepository = new LoanRepository(clients);
     final ItemRepository itemRepository = new ItemRepository(clients, true, true);
     final UserRepository userRepository = new UserRepository(clients);
-
+    final RequestQueueRepository requestQueueRepository = RequestQueueRepository.using(clients);
```

Comment:
This method appears to be for an overridden renewal. There isn't an explicit scenario for this in the story, is this something that you've checked with Sean?

Reply:
It was just my thought, I have not checked it with Sean. Shell I do it?

Reply:
Yes, please check with Sean. It may be that overrides should be disallowed as well.

Reply:
I asked him yesterday in slack but he is silent 😄 , So I left my comment under the ticket. 
Probably he is on vacation.

Reply:
Cate is fine with it, but Sean is still silent...

Reply:
Sean is unlikely to respond until the early afternoon, as he is based in the US.

I think we should wait till late afternoon, and if we haven't heard anything merge this change. It can be undone if necessary.